tests/suite/telemetry_test.go

```diff
@@ -0,0 +1,177 @@
+package suite
+
+import (
+	"fmt"
+	"os/exec"
+	"strings"
+
+	. "github.com/onsi/ginkgo/v2"
+	. "github.com/onsi/gomega"
+	core "k8s.io/api/core/v1"
+	crClient "sigs.k8s.io/controller-runtime/pkg/client"
+)
+
+const (
+	collectorNamespace        = "collector"
+	collectorChartReleaseName = "otel-collector"
+	// FIXME(pleshakov): Find a automated way to keep the version updated here similar to dependabot.
+	collectorChatVersion = "0.73.1"
+)
+
+var _ = Describe("Telemetry test with OTel collector", Label("telemetry"), func() {
+	var skipped bool
+
+	BeforeEach(func() {
+		suiteConfig, _ := GinkgoConfiguration()
+		if suiteConfig.LabelFilter != "telemetry" {
+			skipped = true
+			Skip("Telemetry test should be run separately with 'telemetry' label because of its NGF installation requirements")
```

Comment:
Seems like we'll need two separate functional test runs in our pipeline then. Call `go test` twice? Once for this test, once for all the rest.

Reply:
yeah.
or we change how NGF is deployed, so individual tests have control over when and how